src/com/pentalog/pentastagiu/homework2/shop/ClientCart.java

```diff
@@ -0,0 +1,25 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ClientCart {
+
+    private ClientsOfShop name;
+    ItemsOnSale price;
+    int numbersOfProducts;
+
+    double getTotalPrice(double ...price){
+        double suma=0;
+        for (int i=0; i<numbersOfProducts; i++ ){
+            suma=+price[i];
+        }
+        suma= suma - (suma*name.priceForCostumer()/100);
+        return suma;
+    }
+
+
+
+    public ClientCart(ClientsOfShop name, ItemsOnSale price, int numbersOfProducts) {
```

Comment:
ordinea recomandata intr-o clasa e: fielduri, constructor, getteri/setteri si apoi alte metode
+ nu e nevoie pentru a crea un Cart sa ii spui de la inceput cate produse vor fi in cos, vrem sa fie mai flexibil, sa permitem adaugarea oricator produse si la fel si pretul, nu e ok sa fie setat in constructor